models/DataObject/ClassDefinition/Data/Localizedfields.php

```diff
@@ -883,16 +895,6 @@ public function getMaxTabs(): int
         return $this->maxTabs;
     }
 
-    public function setLabelWidth(int $labelWidth): void
```

Comment:
why is that removed here? 
at least it doesn't match with PRs description...

Reply:
Because the method is also defined in the parent class. The php docs are inherit.